tests/e2e/vsan_stretched_cluster.go

```diff
@@ -447,6 +486,7 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 			fds.hostsDown = nil
```

Comment:
wherever and whatever faults you are inducing, before starting a new test, make a check in AfterEach() block that all your ESXi hosts should be in up and running state and k8s nodes to be in ready state, otherwise if the testbed is in unhealthy state, further test case will going to break

Reply:
done